pyglmnet/pyglmnet.py

```diff
@@ -5,6 +5,7 @@
 
 import numpy as np
 from scipy.special import expit
+from sklearn.base import BaseEstimator, RegressorMixin
```

Comment:
okay, but we can't do this :) we should copy the `BaseEstimator` class ...

unless you want to create a dependency. Let's start simple with just the `BaseEstimator` and see what features we miss from `RegressorMixin`? As far as I remember, `BaseEstimator` should already allow you to do cross-validation ...